backend/installGlobalHook.js

```diff
@@ -57,6 +65,33 @@ function installGlobalHook(window: Object) {
           this._listeners[evt].map(fn => fn(data));
         }
       },
+      // Fiber-only:
+      supportsFiber: true,
+      _fiberRoots: {},
+      getFiberRoots(rendererID) {
+        return this._fiberRoots[rendererID] || (this._fiberRoots[rendererID] = new Set());
+      },
+      onCommitFiberUnmount: function(rendererID, fiber) {
+        // TODO: can we use this for roots too?
+        if (this.helpers[rendererID]) {
+          this.helpers[rendererID].handleCommitFiberUnmount(fiber);
+        }
+      },
+      onCommitFiberRoot: function(rendererID, root) {
+        const mountedRoots = this.getFiberRoots(rendererID);
```

Comment:
It's unfortunate that the use of `this` instead of closure capture invalidates the suggestion I had in https://github.com/facebook/react/pull/8818/files#r98101132